docs/application/dotnet/guides/nlp/overview.md

```diff
@@ -0,0 +1,13 @@
+# Natural Language Processing
+
+The natural language processing features introduce how you can process sentence with Nlp class . They also describe how you can call methods about Nlp in your application.
+
+You can use the following natural language processing features in your .NET applications:
+
+-   [Natural Language Processing](nlp.md)
+
+    You can use natural language processing functionalities in your application, such as language detect, word_tokenize, lemmatize, ne_chunk, and pos_tag. Nlp is a subset of natural language toolkit that specifies an interface and a protocol for basic natural language processin.
+
+## Related Information
+* Dependencies
```

Comment:
May I know whether this dependency is for mobile or wearable?
To maintain consistency, as per other docs, it has to be specified as below:
Dependencies:

- Tizen  4.0 and Higher for Mobile

- Tizen  4.0 and Higher for Wearable

Reply:
We don't divide profile information for **dotnet** guides.
If you check other guide pages under dotnet, you'd see there are no profile information also.

So, this dependencies should keep current status.

Reply:
Ok, understand.